.github/workflows/publish.yml

```diff
@@ -33,14 +33,14 @@ jobs:
 
     - name: Bump react version patch
       run: |
-        grep -E "[0-9]+\.[0-9]+\.[0-9]+-beta\.[0-9]+" ./packages/react-icons/package.json | python3 -c """
+        grep -E "[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+" ./packages/react-icons/package.json | python3 -c """
```

Comment:
Rather than replacing 'beta' with 'rc' in the grep, can you just do it as either beta or rc? Along similar lines, can you just make that bit conditionally match, so it doesn't have to change for a future update that is beta/rc/actual release?